From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nick Turner <nic@amazon.com>
Date: Mon, 14 Mar 2022 14:06:47 -0700
Subject: [PATCH] --EKS-PRIVATE-- Delete expiration cache log statement

Why is patch private and not public/upstreamed?:

If there are plans up upstream, provide relevant links:

Description of changes, customer issues, internal docs, etc.:
---
 staging/src/k8s.io/client-go/tools/cache/expiration_cache.go | 2 --
 1 file changed, 2 deletions(-)

diff --git a/staging/src/k8s.io/client-go/tools/cache/expiration_cache.go b/staging/src/k8s.io/client-go/tools/cache/expiration_cache.go
index dfa95619309..30ace90f87e 100644
--- a/staging/src/k8s.io/client-go/tools/cache/expiration_cache.go
+++ b/staging/src/k8s.io/client-go/tools/cache/expiration_cache.go
@@ -21,7 +21,6 @@ import (
 	"time"
 
 	"k8s.io/apimachinery/pkg/util/clock"
-	"k8s.io/klog/v2"
 )
 
 // ExpirationCache implements the store interface
@@ -99,7 +98,6 @@ func (c *ExpirationCache) getOrExpire(key string) (interface{}, bool) {
 		return nil, false
 	}
 	if c.expirationPolicy.IsExpired(timestampedItem) {
-		klog.V(4).Infof("Entry %v: %+v has expired", key, timestampedItem.Obj)
 		c.cacheStorage.Delete(key)
 		return nil, false
 	}
